addons/website/static/lib/summernote/src/js/EventHandler.js

```diff
@@ -420,7 +420,7 @@ define([
           $btn.parents('.popover').hide();
         }
 
-        if ($.isFunction($.summernote.pluginEvents[eventName])) {
+        if ($.isFunction($.summernote.pluginEvents[eventName]) && eventName != "formatBlock") {
```

Comment:
Please, no modification in summernote (except my changes) as I undertake the remains of the library and create the merge proposal to have our feature and fix in the lib.
If you want to change formatBlock you must look in our files who use $.summernote.pluginEvents. But for this task I think no need to change comportment.